### PR TITLE
Set up pundit

### PR DESCRIPTION
**Pull request by NikitaMasalkov**

### CODE REVIEW

File: app/views/dashboard/index.html.erb [hunk 1]
```diff
@@ -52,28 +52,36 @@
 
 <div class="container pt-5">
   <div class="row">
+  <% unless regular_users.empty? %>
```
Comment (vkalach) [inline app/views/dashboard/index.html.erb]: don't use unless
it's not a good practice when you can use if

File: app/views/dashboard/index.html.erb [hunk 2]
```diff
@@ -52,28 +52,36 @@
...
     <div class="col-md-6">
       Users:
     </div>
+  <% end %>
+  <% unless admin_users.empty? %>
```
Comment (vkalach) [inline app/views/dashboard/index.html.erb]: also, don't use empty
use instead of it blank?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -11,7 +11,7 @@
 
   <body>
     <%= render 'layouts/navbar' %>
-    <div class="content-top-padding">
+    <div class="tp-8">
```
Comment (vkalach) [inline app/views/layouts/application.html.erb]: pt :sweat_smile:

File: app/policies/user_policy.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  class Scope
+    attr_reader :user, :scope
+
+    def initialize(user, scope)
+      @user  = user
+      @scope = scope
+    end
+
+    def resolve
+      if user.present? && user.admin?
```
Comment (vkalach) [inline app/policies/user_policy.rb]: you can use user.&admin?

File: app/policies/user_policy.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    def initialize(user, scope)
+      @user  = user
+      @scope = scope
+    end
+
+    def resolve
+      if user&.admin?
+        scope.all
+      else
+        scope.where(role: 'user')
```
Comment (NikitaMasalkov) [inline app/policies/user_policy.rb]: scope.user

File: app/views/dashboard/index.html.erb [hunk 6]
```diff
@@ -50,30 +50,38 @@
   </ul>
 </div>
 
-<div class="container pt-5">
+<div class="container ptset-up-pundit-5">
```
Comment (vkalach) [inline app/views/dashboard/index.html.erb]: typo

**Pull request merged**